src/blog/emscripten-standalone-wasm.md

```diff
@@ -0,0 +1,180 @@
+---
+title: 'Outside the Web: Standalone WebAssembly Binaries Using Emscripten'
+author: 'Alon Zakai'
+avatars:
+  - 'alon-zakai'
+date: 2019-11-06 16:45:00
+tags:
+  - WebAssembly
+  - tooling
+description: 'Emscripten is switching to the LLVM WebAssembly backend, resulting in much faster link times️ and many other benefits.'
```

Comment:
This seems like a description for another post.

Reply:
Thanks! Fixed.